Sources/Stripe/Client.swift

```diff
@@ -240,27 +295,29 @@ func updateSubscription(
         "items[0][quantity]": String(quantity),
         "payment_behavior": "error_if_incomplete",
         "proration_behavior": "always_invoice",
-      ].compactMapValues { $0 }
-      ))
+      ]
+        .compactMapValues { $0 }
+    )
+  )
 }
 
 public let jsonDecoder: JSONDecoder = {
   let decoder = JSONDecoder()
   decoder.dateDecodingStrategy = .secondsSince1970
+  decoder.keyDecodingStrategy = .convertFromSnakeCase
   return decoder
 }()
-//  |> \.keyDecodingStrategy .~ .convertFromSnakeCase
 
 public let jsonEncoder: JSONEncoder = {
   let encoder = JSONEncoder()
   encoder.dateEncodingStrategy = .secondsSince1970
+  encoder.keyEncodingStrategy = .convertToSnakeCase
   return encoder
 }()
-//  |> \.keyEncodingStrategy .~ .convertToSnakeCase
 
 func stripeRequest<A>(_ path: String, _ method: FoundationPrelude.Method = .get([:])) -> DecodableRequest<A> {
   var request = URLRequest(url: URL(string: "https://api.stripe.com/v1/" + path)!)
-  request.setHeader(name: "Stripe-Version", value: "2019-12-03")
+  request.setHeader(name: "Stripe-Version", value: "2020-08-27")
```

Comment:
Updating to the latest version of the API. Not a lot has changed, though `customer.sources` no longer auto-expands. I've changed this to be an optional and added expansion for the account page.